instfiles/xrdp.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=xrdp daemon
+Documentation=man:xrdp(8) man:xrdp.ini(5)
+Requires=xrdp-sesman.service
+After=network.target xrdp-sesman.service
+
+[Service]
+Type=forking
+PIDFile=/var/run/xrdp.pid
```

Comment:
hardcode

Reply:
I've removed the file as its get build at compile time